M6-Backend/FacilityInterfaces/interfacespecific.js

```diff
@@ -0,0 +1,29 @@
+
+function interfacespecific(numberOfType){
+    switch (numberOfType) {
+        case 0:
+            return "posts";
+            
+        case 1:
+            return  "studys";
+            
+        case 2:
+            return  "entertainments";
+            
+        case 3:
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/CPEN321HELP/CPEN321/pullRequest?prid=9852520)
```suggestion
    case 3:
```